src/routes/LoadSafePage/LoadSafePage.tsx

```diff
@@ -64,13 +66,13 @@ function Load(): ReactElement {
     const ownerEntries = ownerList
       .map((owner) => {
         const ownerFieldName = `owner-address-${owner.address}`
-        const ownerNameValue = values[ownerFieldName]
+        const ownerNameValue = values[ownerFieldName] || values[FIELD_SAFE_OWNER_ENS_LIST][owner.address]
```

Comment:
This "or" should be a function, like the Safe name getter. Because the same logic is used on 3 screens.